index.html

```diff
@@ -13,33 +13,50 @@
 
 <p>The <a href="resources.html">Training Course Material</a> page contains links to material for printing out and using in training sessions, ZIP archives of the PDFs for all the current versions, customised versions of some of the Guides and other resources such as Omnigraffle&reg; and Word&reg; files.</p>
 
-<p>The current release version of OMERO.insight is 5.2.7. You can tell what version of OMERO you are using by looking at the login screen, or selecting <span class="bold">About OMERO.insight</span> from the Help menu in OMERO.insight.</p>
+<p>The current release version of OMERO.insight is 5.3.0. You can tell what version of OMERO you are using by looking at the login screen, or selecting <span class="bold">About OMERO.insight</span> from the Help menu in OMERO.insight.</p>
 
 <p><img src="images/indexVersion.jpg" alt="OMERO.insight Version" style="width: 80%;"/></p>
 
-<p>All the Version 5 material on the User Help website applies to both version 5.1.4 and version 5.2, with the exception of the new features mentioned below, but note that version 5.1 is now unsupported and will not be updated.</p>
+<p>All the Version 5 material on the User Help website applies to version 5.2 and 5.3, with the exception of the new features mentioned below, but note that version 5.2 are now unsupported and will not be updated.</p>
 
 <p>All material is covered by the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 Unported License</a> - you are free to share or adapt content as long as you credit the Open Microscopy Environment. The exception to this is the screenshots and videos which can only be used for non-commercial purposes.</p>
 
 <div class="line-block">
   <div class="line"><br /></div> <!-- end #line -->
 </div><!-- end #line-block -->
 
-<p><img src="images/indexNew.png" alt="New flag image" class="flag-left"/> The Populate Metadata script has been added to the scripts shipped with OMERO. This allows correctly formatted metadata to be attached to High Content Screening (HCS) data after import. For further details see the <a href="scripts.html#metadata">Using Scripts</a> page.</p>
+<p><img src="images/indexNew530.png" alt="New flag image" class="flag-left"/><a href="web-client.html" target="blank">OMERO.iviewer</a> - the Full Viewer in OMERO.web has been completely revamped as a WebApp using new technologies, allowing you to draw ROIs in OMERO.web, and much more. For further details see the <a href="web-client.html">Using OMERO.web</a> page.</p>
```

Comment:
iviewer won't be released until after 5.3.0 (soon after but not at the same time) so i will comment this out when I update this PR